inst/pages/exercises.qmd

```diff
@@ -1101,18 +1101,17 @@ variables (point 3). The results of RDA are usually assessed with PERMANOVA
 exercise. This is a relatively complex procedure, but the way this is broken
 down into steps below will hopefully make more sense.
 
-1. Import the mia and vegan packages, load any of the example data sets
+1. Import the mia package, load any of the example data sets
 mentioned in [@sec-example-data] with `data` and store it into a variable
 named `tse`.
 2. Transform the `counts` assay into relative abundances with `transformAssay`.
-3. Perform RDA with `getRDA` to see how much Diet can explain the relative
+3. Perform RDA with `addRDA` to see how much Diet can explain the relative
 abundance assay (`formula = assay ~ Diet` and `assay.type = relabundance`) in
 terms of Bray-Curtis dissimilarity (`method = "bray"`).
 4. Extract the RDA dimensions from the appropriate reducedDim slot with
 `attr(reducedDim(tse, "RDA"), "rda)` and store it into `rda`.
```

Comment:
Can you still add here `rda` variable as this is not clear what `rda` refers here